### DISCUSSION TITLE
Добавление маршрутизации через VPN

### DISCUSSION THREAD

**Discussion by Garykom** [bol-van/zapret#866] [Q&A]
Хочется часть сетей/ip и доменов автоматически пускать через VPN
Причем с раздельными настройками какие домены/сети через какие интерфейсы и включать ли дурение

Как это можно реализовать?
Без установки отдельного решения а все внутри zapret.

**Comment by bol-van** [comment]
В zapret есть система ipban. Примеры ее использования в docs/redsocks.txt и docs/wireguard_iproute_openwrt.txt
